src/azure-cli/azure/cli/command_modules/containerapp/_transformers.py

```diff
@@ -18,6 +19,37 @@ def transform_containerapp_output(app):
     return result
 
 
+def clean_up_sensitive_values(response_json):
```

Comment:
Hi @p-bouchon 
Please add tests for properties.template.scale.rules[] metadata.

Reply:
added tests for these scenarios